core/src/main/scala/tapir/generic/Configuration.scala

```diff
@@ -0,0 +1,29 @@
+package tapir.generic
+import java.util.regex.Pattern
+
+final case class Configuration(transformMemberName: String => String) {
+  def withSnakeCaseMemberNames: Configuration = copy(
+    transformMemberName = Configuration.snakeCaseTransformation
+  )
+
+  def withKebabCaseMemberNames: Configuration = copy(
+    transformMemberName = Configuration.kebabCaseTransformation
+  )
```

Comment:
is this somehow base on circe? If so, some attribution would be good :) (in a comment or sth like that)

Reply:
:+1: